sportngin/octopolo
# Adding issue command to create issues.

# Description

Its like `pull-request` but for issues.

Upgraded octokit to fix bug with issue creation. 

Sharing some code between them since as far as the GitHub API is concerned, every pull request is an issue, but not every issue is a pull request.

# QA
`bundle exec octopolo issue` - verify issue creation
`bundle exec octopolo pull-request` - verify pr creation
